Solutions/Darktrace/Workbooks/DarktraceWorkbook.json

```diff
@@ -0,0 +1,2628 @@
+              "parameters": [
+                {
+                  "id": "cd64e441-332e-4f47-8602-a25828ebc053",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "aia_type",
+                  "label": "AI Analyst Incident Types",
+                  "type": 2,
+                  "description": "Filter out the types of AI Analyst Incidents available.",
+                  "isRequired": true,
+                  "value": "darktrace_model_alerts_CL | where dtProduct_s =='AI Analyst' | project-rename IncidentName=title_s | project-rename CurrentGroup=externalId_g | project-rename GroupScore=groupScore_d | project-rename SrcDeviceName=bestDeviceName_s | limit 100 | summarize MaxIncidentScore=max(GroupScore), take_any(SrcDeviceName, IncidentName) by CurrentGroup | project MaxIncidentScore, IncidentName, SrcDeviceName, CurrentGroup",
+                  "typeSettings": {
+                    "additionalResourceOptions": [],
+                    "showDefault": false
+                  },
+                  "jsonData": "[\n    {\"value\": \"darktrace_model_alerts_CL | where dtProduct_s =='AI Analyst' | project-rename IncidentName=title_s | project-rename CurrentGroup=externalId_g | project-rename GroupScore=groupScore_d | project-rename SrcDeviceName=bestDeviceName_s | limit 100 | summarize MaxIncidentScore=max(GroupScore), take_any(SrcDeviceName, IncidentName) by CurrentGroup | project MaxIncidentScore, IncidentName, SrcDeviceName, CurrentGroup\", \"label\": \"All\"},\n    {\"value\": \"darktrace_model_alerts_CL | where dtProduct_s =='AI Analyst' | project-rename IncidentName=title_s | project-rename CurrentGroup=externalId_g | project-rename GroupScore=groupScore_d | project-rename SrcDeviceName=bestDeviceName_s | where SrcDeviceName !contains 'SaaS' | limit 100 | summarize MaxIncidentScore=max(GroupScore), take_any(SrcDeviceName, IncidentName) by CurrentGroup | project MaxIncidentScore, IncidentName, SrcDeviceName, CurrentGroup\", \"label\": \"Network\"},\n    {\"value\": \"darktrace_model_alerts_CL | where dtProduct_s =='AI Analyst' | project-rename IncidentName=title_s | project-rename CurrentGroup=externalId_g | project-rename GroupScore=groupScore_d | project-rename SrcDeviceName=bestDeviceName_s | where SrcDeviceName contains 'SaaS' | limit 100 | summarize MaxIncidentScore=max(GroupScore), take_any(SrcDeviceName, IncidentName) by CurrentGroup | project MaxIncidentScore, IncidentName, SrcDeviceName, CurrentGroup\", \"label\": \"SaaS\"}\n]",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 6"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "{aia_type}",
```

Comment:
@JustasZDT, please check and update the query, thanks.

Reply:
@v-sabiraj The KQL for that query is embedded within the selected aia_type, so it shouldn't need updating. We followed this sample online from a Microsoft form: https://techcommunity.microsoft.com/t5/azure-observability/if-statment-in-a-kql-query/m-p/1363551